src/Multipart/File.js

```diff
@@ -447,7 +447,7 @@ class File {
      * Otherwise move the tmpFile to the user specified
      * location.
      */
-    await fs.move(this.tmpPath, path.join(location, options.name))
+    await fs.move(this.tmpPath, path.join(location, options.name), { overwrite: options.overwrite })
```

Comment:
Can we make sure we always pass a boolean

```js
overwrite: !!options.overwrite
```

It doesn't change anything technically, but makes it clear that we are passing a boolean only